maven-plugins/bnd-maven-plugin/src/main/java/aQute/bnd/maven/plugin/AbstractBndMavenPlugin.java

```diff
@@ -254,6 +260,16 @@ public void execute() throws MojoExecutionException, MojoFailureException {
 				throw new MojoExecutionException("Sub-bundles not permitted in a maven build");
 			}
 
+			if (deleteExistingManifest) {
+				// delete existing MANIFEST.MF
+				// e.g. from previous build to avoid that this MANIFEST.MF
+				// becomes input to the bnd process (which could lead to
+				// suprising results in some cases)
+				File existingMetaInf = getManifestPath();
+				logger.info("Delete existing manifest: {}", existingMetaInf);
+				IO.delete(existingMetaInf);
```

Comment:
I think deleting the manifest file is only working when the manifest path points to the default (classes/META-INF/MANIFEST) what is not always the case.

Instead I think the real culprit is some lines below where you create a JAR from the classesDir, this one should be filtered to not include the Manifest-File so `Pattern doNotCopy` might better be specified and remove the unwanted file.

Reply:
@laeubi could you create a new Issue based on the comment with a few more details, describing when this could be a problem?

Reply:
I just wanted to mention that this only solves a very specific case in a very specific way.
It might therefore be better to handle it in a generic way (that is when creating the (bnd) Jar from a folder) and without special options / deletion / ...

Reply:
Ah ok thanks @laeubi Yes, I agree, but in this case I don't feel confident enough to make such a change, because don't have a specific problem and I am too unfamiliar with the BndMavenPlugin ( I don't use it), that I can foresee the consequences. The only reason I made this PR here is, that I just wanted to help out fixing this specific problem for @Marcono1234 and since it was a new option, I felt it it low risk. Otherwise I wouldn't have touched this. 

So if you have a testcase a PR is welcomed. 

@bjhargrave @rotty3000 any opinions?